src/algebraic_geometry/locally_ringed_space.lean

```diff
@@ -46,16 +43,19 @@ namespace LocallyRingedSpace
 
 variables (X : LocallyRingedSpace)
 
+/--
+An alias for `to_SheafedSpace`, where the result type is a `RingedSpace`.
+This allows us to use dot-notation for the `RingedSpace` namespace.
+ -/
+def to_RingedSpace : RingedSpace := X.to_SheafedSpace
+
 /-- The underlying topological space of a locally ringed space. -/
 def to_Top : Top := X.1.carrier
 
 instance : has_coe_to_sort LocallyRingedSpace :=
 { S := Type u,
   coe := λ X : LocallyRingedSpace, (X.to_Top : Type u), }
 
--- PROJECT: how about a typeclass "has_structure_sheaf" to mediate the 𝒪 notation, rather
```

Comment:
I think this is still a good project, right?

Reply:
I think this comment is a little outdated since this notation is never defined elsewhere and is also never used. Personally, I'd rather have it called `LocallyRingedSpace.sheaf`. But this has nothing to do with this PR, I don't know why I removed the comment in this one. I'll add it back in for now.